src/main/resources/org/jenkinsci/plugins/configfiles/ConfigFilesManagement/selectprovider.properties

```diff
@@ -18,4 +18,4 @@
 
 selectprovider=Select the file type you want to create
 filetype=Type
-
+configIdSuffixDescription=Config file ID is prefixed by the provider name (e.g. "org.jenkinsci.plugins.configfiles.maven.MavenSettingsConfig") and suffixed by the given "ID suffix" or by a random number if no "ID Suffix" is specified

```

Comment:
I tried and rolled back because all the logic of the plugin is based on the fact that the ConfigProvider implementation can be deduced from the ID.

Changing this would require more much work.

See 

* [ConfigProvider#isResponsibleFor(String configId)](https://github.com/jenkinsci/config-file-provider-plugin/blob/be5f4c87f3014e3937291112c59663e15ae54588/src/main/java/org/jenkinsci/lib/configprovider/ConfigProvider.java#L86)

* [ConfigFilesManagement#doRemoveConfig(StaplerRequest res, StaplerResponse rsp, @QueryParameter("id") String configId)](https://github.com/jenkinsci/config-file-provider-plugin/blob/dd26c71308ebc4a2bade11fc2ca1831d26c4b5e1/src/main/java/org/jenkinsci/plugins/configfiles/ConfigFilesManagement.java#L231)

      ```java
    public HttpResponse doRemoveConfig(StaplerRequest res, StaplerResponse rsp, @QueryParameter("id") String configId) throws IOException {
        checkPermission(Hudson.ADMINISTER);
        for (ConfigProvider provider : ConfigProvider.all()) {
            if (provider.isResponsibleFor(configId)) {
                provider.remove(configId);
            }
        }
        return new HttpRedirect("index");
    }
```